src/style.css

```
@@ -0,0 +1,11 @@
+.input {
+  margin-bottom: 10px;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
+
+fieldset :last-child {
+  margin: 0;
```

```suggestion
  margin-bottom: 0;
```
Ось тут спробуй прийняти зміни а потім стягнути їх собі через `git pull` якщо хочеш